frontend/views/components/Modal/Modal.vue

```diff
@@ -71,9 +72,13 @@ export default {
     },
     updateUrl () {
       if (this.content) {
-        this.$router.push({ query: { modal: this.content, subcontent: this.activeSubcontent() } })
+        this.$router.push({ query: { modal: this.content, subcontent: this.activeSubcontent() } }).catch(err => {
+          console.log(err)
+        })
       } else {
-        this.$router.push({ query: null })
+        this.$router.push({ query: null }).catch(err => {
+          console.log(err)
```

Comment:
Errors must all be logged with `console.error`. This can be simplified by writing it like this:

```js
.catch(console.error)
```